js/babel-preset-noms/index.js

```diff
@@ -39,8 +38,17 @@ const development = {
     preset('react'),
   ],
   plugins: [
+    plugin('syntax-async-functions'),
+    plugin('syntax-flow'),
+    plugin('transform-async-to-generator'),
+    plugin('transform-class-properties'),
     plugin('transform-es2015-modules-commonjs'),
```

Comment:
The ordering matters. I can play around with it a bit more to see if I can unify the ordering a bit more and if that works, extract the shared plugins to its own array.